tumblr-photo-video-ripper.py

```diff
@@ -196,12 +196,21 @@ def _download_media(self, site, medium_type, start):
                 print("Site %s does not exist" % site)
                 break
 
+            response_file = "{0}/{0}_{1}_{2}_{3}.response.xml".format(site, medium_type, MEDIA_NUM, start)
+            with open(response_file, "w") as text_file:
+                text_file.write(response.content)
+
             try:
                 xml_cleaned = re.sub(u'[^\x20-\x7f]+',
                                      u'', response.content.decode('utf-8'))
                 data = xmltodict.parse(xml_cleaned)
                 posts = data["tumblr"]["posts"]["post"]
                 for post in posts:
+                    post_json_file = "{0}/{0}_post_id_{1}.post.json".format(site, post['@id'])
+                    print(post_json_file)
```

Comment:
No need for this kind of print I think.

Reply:
Good point! Sorry I overlooked it when making push request, I just do this on my own. Removing from this PR (pull request)